package-res/resources/web/prompting/components/DojoDateTextBoxComponent.js

```diff
@@ -213,6 +213,10 @@ define(['cdf/components/BaseComponent', "dojo/date/locale", 'dijit/form/DateText
             return true;
           }
 
+          if(dateFormat.match(/mm/)) {
+            return true;
+          }
+
```

Comment:
Isn't this too generic ? I thought we were going to focus on the specific yy-mm-dd case. 
Doesn't this break use cases where we are trying to show minutes instead of months ?